packages/tkg-autoscaler/README.md

```diff
@@ -0,0 +1,25 @@
+# tkg-clusterclass-aws Package
+
+This package provides << awesome functionality >> using [tkg-autoscaler](https://INFO_NEEDED).
```

Comment:
> << awesome functionality >>

I like this 😸

Reply:
Should we elaborate though? Even if it's very few concrete words, it'd still be better.